src/App.tsx

```diff
@@ -1,61 +1,114 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
 export const App = () => {
+  const getUserById = (id: number) => {
+    const search = usersFromServer.find(usr => usr.id === id) as User;
+
+    if (!search) {
+      throw new Error('user not found');
+    }
+
+    return search;
+  };
+
+  const [selectedUser, setSelectedUser] = useState(0);
```

Comment:
The `selectedUser` state is initialized with `0`, which represents 'no user selected'. It might be clearer to use `null` or `undefined` to represent this state, as `0` could be a valid user ID in some contexts.